commit d69056fcc4f436ea60961889b2b55829b453ea31
Author: David Forster <dforster@brocade.com>
Date:   Fri Feb 24 14:20:32 2017 +0000

    vti6: return GRE_KEY for vti6
    
    [ Upstream commit 7dcdf941cdc96692ab99fd790c8cc68945514851 ]
    
    Align vti6 with vti by returning GRE_KEY flag. This enables iproute2
    to display tunnel keys on "ip -6 tunnel show"
    
    Signed-off-by: David Forster <dforster@brocade.com>
    Signed-off-by: David S. Miller <davem@davemloft.net>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1375_d69056fcc4f436ea60961889b2b55829b453ea31.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1375_d69056fcc4f436ea60961889b2b55829b453ea31.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
